src/resources/smells.js

```diff
@@ -0,0 +1,24 @@
+const Smells = [
+	{
+		name: 'LINE_TOO_LONG',
+		title: 'Line too long',
+		description: 'Lines that are too long make your code less readable',
+		slug: 'line-too-long',
```

Comment:
Could you tell me if we need name and slug? I think both fields are pretty much the same.

Reply:
Slug is used for routing (i.e. `/knowledge-base/smell-slug`) and fetching appropriate markdown file (i.e. `smell-slug.md`). Slugs may be helpful for SEO - they can be longer and thus more descriptive than smell names.